include/RAJA/util/Operators.hpp

```diff
@@ -386,7 +386,7 @@ struct logical_and : public detail::comparison_function<Arg1, Arg2>,
   {
     return lhs && rhs;
   }
-  static constexpr const bool identity = true;
+  static constexpr bool identity() {return true;}
```

Comment:
I was on two telecons at the same time earlier today.  Granted, I time-sliced the audio, because I've tried to do that in the past and it doesn't work very well.  The human brain is much better at visual concurrency than aural concurrency.